packages/mds-ingest-service/repository/entities/device-entity.ts

```diff
@@ -0,0 +1,41 @@
+import { Entity, Column } from 'typeorm'
+import { IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { DeviceDomainModel } from '../../@types'
+
+export interface DeviceEntityModel extends IdentityColumn, RecordedColumn {
+  device_id: DeviceDomainModel['device_id']
+  provider_id: DeviceDomainModel['provider_id']
+  vehicle_id: DeviceDomainModel['vehicle_id']
+  type: DeviceDomainModel['type']
+  propulsion: DeviceDomainModel['propulsion']
+  year: DeviceDomainModel['year']
+  mfgr: DeviceDomainModel['mfgr']
+  model: DeviceDomainModel['model']
+}
+
+@Entity('devices')
+export class DeviceEntity extends IdentityColumn(RecordedColumn(class {})) implements DeviceEntityModel {
+  @Column('uuid', { primary: true })
+  device_id: DeviceEntityModel['device_id']
+
+  @Column('uuid', { nullable: false })
+  provider_id: DeviceEntityModel['provider_id']
+
+  @Column('varchar', { length: 255, nullable: false })
```

Comment:
`nullable: false` is the default. Makes nullable properties easier to see if the default is omitted.